src/transformers/file_utils.py

```diff
@@ -133,7 +133,7 @@
 MODEL_CARD_NAME = "modelcard.json"
 
 
-MULTIPLE_CHOICE_DUMMY_INPUTS = [[[0], [1]], [[0], [1]]]
+MULTIPLE_CHOICE_DUMMY_INPUTS = [[[0, 1, 2, 3], [1, 2, 3, 4]] * 2]
```

Comment:
Funnel Transformer model pools twice, so it needs a sequence length of 4 to work properly.